packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -115,7 +115,12 @@ export const transformPlotlyJsonToDonutProps = (
 
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33533&commitId=85059cba49e01e68615ae5aef6070004a7821975&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Bottom right edge.chromium.png|1596| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Left center.chromium.png|2616| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic RTL.chromium.png|372| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>